### PR TITLE
Update scaffolding

### PR DESCRIPTION
**Pull request by anoblet**
Progress on #45

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def index_i18n(lang):
     if lang not in SUPPORTED_LANGS:
         abort(404)
 
-    return render_template('%s/index.html' % lang)
+    return render_template('%s/splash.html' % lang)
```
Comment (rviscomi) [inline src/main.py]: Need to also update the `index` method above.

File: src/templates/en/splash.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{% extends "en/base.html" %}
```
Comment (rviscomi) [inline src/templates/en/splash.html]: Also create a `ja/splash.html`

Comment (anoblet) [reply src/templates/en/splash.html]: Updated

File: src/main.py (hunk 1 above)
Comment (anoblet) [reply src/main.py]: Updated

**Pull request merged**